README.md

```diff
@@ -66,7 +66,7 @@ $ bower install angularfire --save
 
 * [Quickstart](docs/quickstart.md)
 * [Guide](docs/guide/README.md)
-* [API Reference](https://angularfire.firebaseapp.com/api.html)
+* [API Reference](docs/reference.md)
```

Comment:
> The /s work fine though, check out this file and try the reference link :)

In that case, is this one incorrect? Shouldn't it also begin with a /? Sigh, I feel like a GitHub noob suddenly. I didn't know I was so poorly versed in how URLs work.